tests/decimal_tests.rs

```diff
@@ -3514,6 +3514,36 @@ fn to_from_sql() {
     }
 }
 
+#[cfg(feature = "db-postgres")]
+#[test]
+fn from_sql_special_numeric() {
```

Comment:
Same thing here - (it filters postgres test by those that have postgres in the name):
```suggestion
fn postgres_from_sql_special_numeric() {
```